src/libs/actions/CompanyCards.ts

```diff
@@ -791,6 +792,18 @@ function openAssignFeedCardPage(policyID: string, feed: CompanyCardFeed, workspa
     API.read(READ_COMMANDS.OPEN_ASSIGN_FEED_CARD_PAGE, parameters, {optimisticData, successData, failureData});
 }
 
+function openPolicyAddCardFeedPage(policyID: string | undefined) {
+    if (!policyID) {
+        return;
+    }
+
+    const parameters: OpenPolicyAddCardFeedPageParams = {
+        policyID,
+    };
+
+    API.read(READ_COMMANDS.OPEN_POLICY_ADD_CARD_FEED_PAGE, parameters);
```

Comment:
I think this will need to be `API.write` since this will create the workspaceAccountID if it doesn't already exist.